dataactcore/migrations/versions/ee7bff1d660c_fileE_add_legal_entity_name.py

```diff
@@ -0,0 +1,33 @@
+"""Adding column lega entity name schema v1.2 File E
```

Comment:
Same as the F file one, but this one can be more specific since it's just one column (somehow GitHub didn't keep this comment last time)

Reply:
Updated description and file name